vhost-device-input/Cargo.toml

```diff
@@ -0,0 +1,36 @@
+[package]
+name = "vhost-device-input"
+version = "0.1.0"
+authors = ["Leo Yan <leo.yan@linaro.org>"]
+description = "vhost input backend device"
+repository = "https://github.com/rust-vmm/vhost-device"
+readme = "README.md"
+keywords = ["input", "vhost", "virt", "backend"]
```

Comment:
I think we should probably coordinate this with the other crates. I also wonder whether we should maybe add some 'rust-vmm' keyword... But I did not bring it up here since the other daemons also does not add it.

Reply:
I personally think `rust-vmm` would be good to be added.  I will update this in the new version patch.